### ISSUE TITLE
[Order Creation] Create Single Product selection view

### ISSUE THREAD

**Issue by ThomazFB** [woocommerce/woocommerce-android#5277]
Once we have the Products section we need to display a view capable of listing every existing non-variable product inside that given store when the `Add Product` button is clicked inside the Order Creation form.

Please note that the search bar will be implemented afterward on another issue.

Depends on #5276

![image](https://user-images.githubusercontent.com/5920403/142503471-b19eedb7-10f7-4bac-a0a8-018f25dbd260.png)
